templates/takeovers/_compliance-webinar.html

```diff
@@ -0,0 +1,26 @@
+<section lang="en" data-lang="{{ locale }}" class="p-strip--image is-deep js-takeover is-dark p-takeover--compliance {% if not default %}u-hide{% endif %}"
+    {% if default %}data-default="true" {% endif %}>
+    <div class="row u-vertically-center">
+        <div class="col-7 u-fade-left--medium">
```

Comment:
Where is the styling for `u-fade-left--medium`? If it's in another takeover. You should for that here to decouple them.